### ISSUE TITLE
Add high contrast and e-paper display themes.

### ISSUE THREAD

**Issue by DrewNaylor** [DrewNaylor/Retiled#246]
They'll just be based off the currently-existing themes, but specifically changed to:
1. Be as high-contrast as possible, by using black and white wherever possible (I need to check what colors the Action Center would have in high contrast, because I never looked at it like that)
2. Replace accent colors with specified colors in both light and dark themes. So for buttons, they would have white as the fill color in the dark theme when pressed instead of an accent color. Some things might not be able to just switch out the accent color for white or black, so I'll also have to have borders appear on items like tiles I think.

There should also be a version of the high contrast themes that aren't necessarily as high-contrast if it's too intense like with all the extra borders. The main idea for this is like e-paper displays. Options to reduce how many animations play also need to be implemented for both accessibility reasons and so they don't look bad on e-paper displays. I also have an idea to add accent colors specifically meant for e-paper displays, but they might just be greys mostly.
